### PR TITLE
Remove travis

### PR DESCRIPTION
**Pull request by serathius**
Travis CI is broken for some time and was already removed from 3.4
branch.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,51 +0,0 @@
...
-  - "1.16.3"
-  - tip
-
-notifications:
-  on_success: never
-  on_failure: never
-
-env:
-  matrix:
-    - TARGET=linux-amd64-coverage
```
Comment (ptabor) [inline .travis.yml:18]: Can we migrate coverage ?
Was it blocked on 30min timeout ?

**Review by ptabor — CHANGES_REQUESTED** [review]
We need to recover the coverage tests (or at least have a bug tracing this).

**Comment by hexfusion** [comment]
I think this breaks release tooling as well as a reminder

**Comment by serathius** [comment]
I think we can revisit this PR as Travis is no longer enabled on Etcd repository.

**Comment by serathius** [comment]
cc @spzala @ptabor @ahrtr

**Pull request merged**